data/data-pipeline/data_pipeline/etl/score/etl_score.py

```diff
@@ -533,6 +544,18 @@ def _prepare_initial_df(self) -> pd.DataFrame:
                 logger.info(
                     f"Dropping {len(drop_tracts)} tracts from Linguistic Isolation"
                 )
+            # elif (
```

Comment:
Is there a good reason to have the commented code here?

Reply:
I originally left it in case we wanted to / had to re-implement it... but it's fast enough to do that I deleted it to keep things clean. Ty!